src/items/FilePdf.tsx

```diff
@@ -24,25 +24,14 @@ const FilePdf: FC<FilePdfProps> = ({
 }) => {
   const embedRef = useRef<HTMLEmbedElement>(null);
   const classes = useStyles();
-  const [height, setHeight] = useState<number | string>(
-    defaultHeight ?? '100%',
-  );
-
-  const onLoad: ReactEventHandler<HTMLEmbedElement> = (e) => {
-    // set pdf height -> probably very high
-    const newHeight = (e.target as HTMLEmbedElement)?.offsetParent
-      ?.scrollHeight;
-    newHeight && setHeight(newHeight);
```

Comment:
Does your implementation dynamically set the height? I think since you're wrapping the pdf in a new container, it doesn't get the right parent for the height. Would it be possible to get a height from the window maybe instead?